### ISSUE TITLE
MSE value

### ISSUE THREAD

**Issue by pranithsrujanroy** [Rajat-dhyani/Stock-Price-Predictor#4]
You claim to have achieved that value of MSE but rather using model.evaluate, calculate the RMSE using the formula and see it is 127.028, no where the real value.
model.evaluate returns the test loss in the last iteration here
see the documentation of [keras.models.Model](https://keras.io/models/model/)
